coda/coda_mdstore/tests/test_views.py

```diff
@@ -641,10 +661,26 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLListScrape(request, 'ark:/00001/id')
 
-    @pytest.mark.xfail(reason='pairtreeCandidateList is not available '
-                              'in scope.')
-    def test_response_content(self):
-        assert 0
+    def test_response_content(self, rf):
+        bag = FullBagFactory.create()
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        request = rf.get('/')
+        text = """<html>
+                    <body>
+                    <tr> <td>test</td> <td>data</td> </tr>
+                    <tr> <td>of </td> </tr>
+                    <tr> <td>
+                        <a href='test.com'>url</a>
+                        <a href='testurl.com'>here</a>
+                    </td> </tr>
+                    </body>
+                  </html>"""
+
+        with mock.patch('coda_mdstore.presentation.urllib.request.urlopen', return_value=text):
+            response = views.bagURLListScrape(request, bag.name)
```

Comment:
Regarding testing `bagURLListScrape` it looks to me like we aren't actually using it and it and  `pairtreeCandidateList` could be removed from the code. I was comparing this test to `bagURLListScrape` and noticed that `bagURLListScrape` calls `getFileHandle(identifier, "", pairtreeCandidateList)`. This should error if it were being used because `getFileHandle` only takes 2 arguments, not 3. I see that `getFileHandle` was previously mocked and unfortunately the way it was done hides that this function is broken. If you agree it can be safely removed, we should make it go away :).

Reply:
Yea...I don't see those views are being used and I think they can be removed.

Reply:
I believe this test can be deleted.

Reply:
I deleted this test and its corresponding view and merged it to master. Once this branch gets merged to py2to3 branch I'll do a rebase.